src/generic_ui/scripts/ui.ts

```diff
@@ -524,7 +524,7 @@ export class UserInterface implements ui_constants.UiApi {
   }
 
   public parseUrlData = (url:string) : { type:social.PeerMessageType; message:string } => {
-    var match = url.match(/https:\/\/www.uproxy.org\/(request|offer)\/(.*)/)
+    var match = url.match(/https:\/\/www.uproxy.org\/(request|offer)\/#?(.*)/)
```

Comment:
This is just changing it so that the `#` will not get included in the match group